templates/default/apache2.conf.erb

```diff
@@ -40,10 +40,10 @@ MaxKeepAliveRequests <%= node['apache']['keepaliverequests'] %>
 #
 KeepAliveTimeout <%= node['apache']['keepalivetimeout'] %>
 
-#<IfModule unixd_module>
-User <%= node['apache']['user'] %>
-Group <%= node['apache']['group'] %>
-#</IfModule>
+<IfModule unixd_module>
+  User <%= node['apache']['user'] %>
+  Group <%= node['apache']['group'] %>
+</IfModule>
 
```

Comment:
👍 This looks good for apache 2.4+ as [User and Group](https://httpd.apache.org/docs/2.4/mod/mod_unixd.html#user) are now part of `unixd`. The reason it was commented out was because on apache 2.2, the directives `User` and `Group` where part of core and there was no `unixd` module so this would configuration change would have failed.